### PR TITLE
Removed empty constructor as developers should use VectorX.ZERO for zero'ed vector

### PR DESCRIPTION
**Pull request by thehutch**
Signed-off-by:thehutch th3hutch@yahoo.co.uk

### CODE REVIEW

File: math/src/test/java/org/spout/math/test/vector/Vector2Test.java [hunk 1]
```diff
@@ -416,7 +411,7 @@ public void testCloning() {
 	@Test
 	public void testCreateDirectionFromRandom() {
 		Random random = new Random();
-		for (int i = 0; i < 10; i++) {
+		for (int i = 0 ; i < 10 ; i++) {
```
Comment (kitskub) [inline math/src/test/java/org/spout/math/test/vector/Vector2Test.java:414]: I'm pretty sure Spout formatting is for (int i = 0; i < 10; i++) {

Comment (thehutch) [inline math/src/test/java/org/spout/math/test/vector/Vector2Test.java:414]: Didn't mean to change this :P I didn't know I had until I made the PR

Comment (kitskub) [inline math/src/test/java/org/spout/math/test/vector/Vector2Test.java:414]: Probably should be fixed.

File: engine/src/main/java/org/spout/engine/filesystem/resource/loader/CubeMeshLoader.java [hunk 2]
```diff
@@ -41,87 +41,83 @@ public CubeMeshLoader() {
 	}
 
 	private static OrientedMesh loadObj(InputStream stream) {
-		Scanner scan = new Scanner(stream);
```
Comment (kitskub) [inline engine/src/main/java/org/spout/engine/filesystem/resource/loader/CubeMeshLoader.java:44]: Is this breaking or fixing?

Comment (thehutch) [inline engine/src/main/java/org/spout/engine/filesystem/resource/loader/CubeMeshLoader.java:44]: It's a convert to try-with-resources

Comment (kitskub) [inline engine/src/main/java/org/spout/engine/filesystem/resource/loader/CubeMeshLoader.java:44]: Okay

**Pull request closed without merging**